properties.go

```diff
@@ -0,0 +1,50 @@
+package httpshopify
+
+import "github.com/MOHC-LTD/shopify/v2"
+
+// PropertyDTOs is a collection of property DTOs.
+type PropertyDTOs []PropertyDTO
+
+// ToShopify converts this DTO to the Shopify equivalent.
+func (dtos PropertyDTOs) ToShopify() []shopify.Property {
+	properties := make([]shopify.Property, 0, len(dtos))
+	for _, dto := range dtos {
+		properties = append(properties, dto.ToShopify())
+	}
+
+	return properties
+}
+
+// PropertyDTO holds custom information for an entity.
+type PropertyDTO struct {
+	// Name is the name of the property.
+	Name string `json:"name"`
```

Comment:
This formatted... looks a little weird.

Reply:
Must be GitHub, it's the correct spacing :thinking:

Reply:
So dto properties don't usually have comments on so removed them, aligns fine now.

Reply:
Oh that was why then.

Reply:
Probably does not need them anyway. Obvious names.